chpl/chpl-service/src/main/java/gov/healthit/chpl/manager/impl/DeveloperManagerImpl.java

```diff
@@ -256,10 +256,6 @@ public DeveloperDTO update(final DeveloperDTO updatedDev, final boolean doValida
                 && !resourcePermissions.isUserRoleAdmin() && !resourcePermissions.isUserRoleOnc()) {
             String msg = msgUtil.getMessage("developer.statusChangeNotAllowedWithoutAdmin");
             throw new EntityCreationException(msg);
-        } else if (!currDevStatus.getStatus().getStatusName().equals(DeveloperStatusType.Active.toString())
```

Comment:
I see that you reverted but I guess I'm still confused about this bit being deleted... the comment two lines down says "if the developer is not active and not going to be active only its current status can be updated" and then the updateStatusHistory method is called. Where does that happen in the updated code? Was it not supposed to be in there?